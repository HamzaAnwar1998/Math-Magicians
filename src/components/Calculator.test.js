import React from 'react';
import renderer from 'react-test-renderer';
// import Calculator from './Calculator';

test('Calculator page should match the text', () => {
  const tree = renderer.create(<h3>Lets do some maths!</h3>).toJSON();
  expect(tree).toMatchSnapshot();
});
