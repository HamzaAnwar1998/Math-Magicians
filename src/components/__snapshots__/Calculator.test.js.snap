// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator page should match the text 1`] = `
<h3>
  Lets do some maths!
</h3>
`;
